priv/repo/migrations/20161019110737_create_donation_goal.exs

```diff
@@ -0,0 +1,18 @@
+defmodule CodeCorps.Repo.Migrations.CreateDonationGoal do
+  use Ecto.Migration
+
+  def change do
+    create table(:donation_goals) do
+      add :title, :string, null: false
+      add :description, :text
+      add :amount_cents, :integer
```

Comment:
I think we should append `_cents`. Stripe doesn't do this, so no reason we should. We should just expect that `amount` is in cents and document it as such.